src/app/docs/vertical-tabs/vertical-tabs-docs.component.html

```diff
@@ -0,0 +1,272 @@
+<sky-docs-demo-page moduleName="SkyTabsModule" moduleSourceCodePath="src/app/public/modules/vertical-tabset/"
+  pageTitle="Vertical tabs">
+  <sky-docs-demo-page-summary>
+    Vertical tabs divide and categorize page content when the number of categories is large.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-demo>
+    <sky-docs-demo-control-panel (selectionChange)="onDemoSelectionChangeTabs($event)">
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-checkbox label="Include counts" propertyName="counts" [checked]="false">
```

Comment:
The *Reset* option for both demos doesn't reset the selected state and collapsed/expanded state to its setting when the demo originally loaded. It only seems to reset whether to include the counts, but I'm pretty sure demos are supposed to reset everything back to the original state.

Reply:
@Blackbaud-SteveBrush and @Blackbaud-SomaliaJamall, do either of you remember what we changed in the other responsive demos for this behavior?

Reply:
There’s a `(reset)` event that can be subscribed to, where the developer can then reset all the values needed. It’s not a fully automatic process.

Reply:
Looks like this is working as expected now. Thanks @Blackbaud-SomaliaJamall!